src/components/table/table-row/modern-row.style.js

```diff
@@ -0,0 +1,36 @@
+import StyledTableCell from '../table-cell/table-cell.style';
+
+function applyModernRowStyling({ colors, table }) {
+  return `   
+    ${StyledTableCell} {
+      padding-top: 0px;
+      padding-bottom: 0px;
+      background-color: ${colors.white};
+    }
+    &:hover {
+      ${StyledTableCell} {
+        background-color: ${table.primary};
+      }
+    }
+  `;
+}
+
+function applyModernSelectedStyling({ table }) {
+  return `
+    background-color: ${table.selected};
+    border-bottom-color: ${table.selected};
+  `;
+}
+
+function applyModernHighlightStyling({ table }) {
+  return `
+    background-color: ${table.selected};
+    border-bottom-color: ${table.selected};
+  `;
+}
```

Comment:
this method does the same thing as the one above

Reply:
Yeah I hadn't merged in the updates in the base, the duplication is removed now